doc/pmempool/pmempool-sync.1.md

```diff
@@ -77,7 +77,8 @@ be opened are recreated.=e=)
 
 `-b, --bad-blocks`
 
-: Fix bad blocks - requires creating or reading special recovery files.
+: Fix bad blocks - requires the super user privileges and causes creating
```

Comment:
requires access to NVDIMM region devices (?) which are root only by default. I would even provide the paths which you can `chmod`.

something like that?